spec/support/download_helpers.rb

```diff
@@ -1,11 +1,13 @@
+# frozen_string_literal: true
+
 module DownloadHelpers
   TIMEOUT = 10
   PATH    = Pathname.new(File.expand_path('../../downloads', __dir__))
 
-  extend self
+  module_function
```

Comment:
after some quick reading up about `extend self` vs `module_function` (including a [rubocop issue about this rule](https://github.com/rubocop-hq/ruby-style-guide/issues/556)), i'm not sure i understand this change well enough to approve it.  but everything else LGTM.